default/scripting/buildings/STARLANE_BORE.focs.py

```diff
@@ -0,0 +1,60 @@
+from buildings.buildings import BORE_POSSIBLE, DO_STARLANE_BORE, SPECIES_LIKES_OR_DISLIKES_BUILDING_STABILITY_EFFECTS
+from focs._effects import (
+    Destroy,
+    EffectsGroup,
+    GenerateSitRepMessage,
+    IsSource,
+    Object,
+    Planet,
+    Source,
+    Target,
+)
+from macros.base_prod import BUILDING_COST_MULTIPLIER
+
+try:
+    from focs._buildings import *
+except ModuleNotFoundError:
+    pass
+
+BuildingType(  # type: ignore[reportUnboundVariable]
+    name="BLD_STARLANE_BORE",
+    description="BLD_STARLANE_BORE_DESC",
+    buildcost=(200 + 50 * Target.System.NumStarlanes) * BUILDING_COST_MULTIPLIER,
+    buildtime=1 + Target.System.NumStarlanes,
```

Comment:
Change the order, so it will solve unneeded cost.

Reply:
It looks the same for me.   Reversed version looks better.

```diff
-    buildtime = 1 + Target.System.NumStarlanes
+    buildtime = Target.System.NumStarlanes + 1
```

I suggest merging it as is and dealing with the dump later.

Reply:
It's not just dump, it's additional operations to execute.

Reply:
It's not just dump, it's additional operations to execute.